nasa-gibs/worldview
# WV 2079 and WV 2007: remove HLS version 1.5 and update Aerosol type layer group

## Description

WV-2079 and WV-2077
Removed references to HLS v1.5 and changed layergroup for VIIRS Aerosol Type to "Aerosol Type" from "Aerosol Optical Depth"

## How To Test


1. Open with /?l=VIIRS_SNPP_Aerosol_Type_Deep_Blue_Best_Estimate&lg=true&t=2021-12-17-T18%3A16%3A18Z
2. Make sure layer group is "Aerosol Type"

1. Open HLS layers: /?v=-215.72180664663668,-133.81255687125724,194.90319335336332,108.90619312874277&l=HLS_S30_Nadir_BRDF_Adjusted_Reflectance,HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg=true&t=2021-12-02-T18%3A16%3A18Z
2. Make sure layer descriptions no longer reference v1.5 in the text or the references section.
3. <v1.5 will also be removed from data download once GIBS merges the fixes on their end>
